episodes/tool.nix

```
@@ -0,0 +1,17 @@
+{
+  lib,
+  buildGoModule,
+}:
+buildGoModule rec {
+  pname = "tool";
+  version = "0.0.1";
+  src = builtins.path {
+    path = ./.;
+    name = "tool";
+    filter = path: type:
+      lib.any (lib.flip lib.hasSuffix path) ["go.mod" "go.sum" ".go"];
+  };
```

```suggestion
  src = lib.fileset.toSource {
    root = ./.;
    fileset = lib.fileset.unions [
      ./go.mod
      ./go.sum
      ./main.go
      ./oauth.go
    ];
  };
```

We can use some basic file set functions now! Filtering for files is still [in review](https://github.com/NixOS/nixpkgs/pull/260265) but should soon also become available :)